src/client/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -75,7 +75,8 @@ internal bool IsCommonOrOrganizationsTenant()
 
         internal static bool IsCommonOrOrganizationsTenant(string tenantId)
         {
-            return !string.IsNullOrEmpty(tenantId) && 
+            return !string.IsNullOrEmpty(tenantId) &&
+                   !IsConsumers(tenantId) &&
```

Comment:
Nice investigation in #5184 to find out this regression was introduced by #3517. Strangely, [that change in #3517 received an explicit upvote](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3517/files#r932451254), so, it seemed like an intentional and even desirable removal. I don't remember any context of this change off the top of my head.